runtime/src/bank_forks.rs

```diff
@@ -44,6 +44,35 @@ impl ReadOnlyAtomicSlot {
     }
 }
 
+/// Convenience type since often root/working banks are fetched together.
+#[derive(Clone)]
+pub struct SharableBanks {
+    root_bank: SharableBank,
+    working_bank: SharableBank,
+}
+
+impl SharableBanks {
+    pub fn root(&self) -> Arc<Bank> {
+        self.root_bank.load()
+    }
+
+    pub fn working(&self) -> Arc<Bank> {
+        self.working_bank.load()
+    }
+
+    pub fn load(&self) -> BankPair {
+        BankPair {
+            root_bank: self.root(),
+            working_bank: self.working(),
+        }
+    }
+}
+
+pub struct BankPair {
```

Comment:
used a struct here simply so the return type of `load` has named fields instead of a tuple where order could get confused.